bandwidth/utilities/web_rtc.py

```diff
@@ -0,0 +1,49 @@
+from bandwidth.model.bxml.response import Response
+from bandwidth.model.bxml.verbs import *
+
+
+def generate_transfer_verb(device_token: str, voice_call_id: str, sip_uri: str = 'sip:sipx.webrtc.bandwidth.com:5060') -> str:
+    """Returns the Transfer verb to perform the SIP transfer without the Response wrapper
+
+    Args:
+        device_token (str): The device token.
+        voice_call_id (str): The Bandwidth voice Call ID.
+        sip_uri (str, optional): The SIP URI to transfer the call to. Defaults to 'sip:sipx.webrtc.bandwidth.com:5060'.
+
+    Returns:
+        str: <Transfer> BXML Verb
+    """
+    uui = "".join(voice_call_id.split("-")[1::])
```

Comment:
You could make a private helper to generate the Transfer and reuse it in this function and in generate_transfer_bxml